devtools/cli/src/main/java/io/quarkus/cli/Encryptor.java

```diff
@@ -0,0 +1,60 @@
+package io.quarkus.cli;
+
+import java.nio.ByteBuffer;
+import java.nio.charset.StandardCharsets;
+import java.security.MessageDigest;
+import java.security.SecureRandom;
+import java.util.Base64;
+import java.util.concurrent.Callable;
+
+import javax.crypto.Cipher;
+import javax.crypto.spec.GCMParameterSpec;
+import javax.crypto.spec.SecretKeySpec;
+
+import io.quarkus.cli.common.OutputOptionMixin;
+import picocli.CommandLine;
+import picocli.CommandLine.Command;
+import picocli.CommandLine.Option;
+
+@Command(name = "encryptor", header = "Encrypt Secrets in AES/GCM/NoPadding")
+public class Encryptor implements Callable<Integer> {
+    @Option(required = true, names = { "-s", "--secret" }, description = "Secret")
+    String secret;
+
+    @Option(required = true, names = { "-k", "--encryption-key" }, description = "Encryption Key")
+    String encryptionKey;
+
+    @Option(hidden = true, names = { "-a", "--algorithm" }, description = "Algorithm", defaultValue = "AES")
```

Comment:
Should we have `algorithm` only ? `mode`/`padding`, if they can vary, they would be encoded in the algorithm name. Having an `algorithm` only will also help with plugging in your Jasypt code from smallrye-config

Reply:
We can add them back if ever proves necessary

Reply:
These are hidden, so they are not visible (or required) when you run the command. We may want to support additional combinations in the future.

They are separate, because you need the algorithm for `Cipher` and `SecretKeySpec`, but we can remove the options and hardcoded them (like the original utils).

Reply:
@radcortez Sure, you are right they can vary, but what I was saying is that it varies in the standard algorithm name, example, `AES/ECB/PKCS5Padding`, `AES/GCM/NoPadding`, or may be some users will want to use PBE based encryption. But since they are hidden for now, it is fine, it will let us revisit it and finalize when the actual requirement to use something else is opened.